text/000-css-literals.md

```diff
@@ -223,26 +223,29 @@ The conversion to prefix and `kebab-case` happens automatically.
 **Code:**
 ```typescript
 export const myCss = css({
-  $myGlobalVariable: "purple",
-  color: "$myGlobalVariable"
+  $myCssVariable: "purple",
+  color: "$myCssVariable",
+  backgroundColor: "$myOtherVariable(red)"
```

Comment:
I don't have better idea right now, but maybe there could be other best way to express this.
However It's also good, so for now maintain this, and if we find better way, let's fixing it.